.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: KI6Kzi4IrRkQ81xH8zkzcwrDja01bIFRd
```

Comment:
This should not be here

consider `repo_token: COVERALLS_REPO_TOKEN`

`COVERALLS_REPO_TOKEN` could be an environmental variable in our CI tool (Travis in our case)

Reply:
just keep it handy so that whenever #4  is merged, we would add it as a variable to travis

Reply:
Noted with thanks